http/src/main/resources/reference.conf

```diff
@@ -17,39 +17,40 @@ dataproc {
     "https://www.googleapis.com/auth/userinfo.profile",
     "https://www.googleapis.com/auth/cloud.useraccounts.readonly",
     "https://www.googleapis.com/auth/bigquery"
-  ],
-  # TODO: temp 2019-07-08
-  welderDisabledNotebooksDir = "/home/jupyter-user"
-  welderEnabledNotebooksDir = "/home/jupyter-user/notebooks"
+  ]
   # Unset to use project-level defaults. Note a VPC/subnet with the literal name "default" may not exist.
   #vpcNetwork = "default"
   #vpcSubnet = "default"
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-2-79-debian9-5e0248a"
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-5e0248a"
 
+  dataprocReservedMemory = 4.5g
+}
+
+image {
+  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:60e28bc"
+  jupyterImage = "us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da"                  # updated 8/26/19
+
+  jupyterImageRegex = "us.gcr.io/broad-dsp-gcr-public/([a-z0-9-_]+):(.*)"
+  rstudioImageRegex = "us.gcr.io/anvil-gcr-public/([a-z0-9-_]+):(.*)"
+  dockerhubImageRegex = "broadinstitute/([a-z0-9-_]+):(.*)"
+}
+
+welder {
```

Comment:
I decided to break some things into a new `welder` structure (the diff looks a little odd).